app/uk/gov/hmrc/gform/validation/ValidationService.scala

```diff
@@ -250,7 +271,7 @@ class ComponentsValidator(
     concreteDate: ConcreteDate,
     offset: OffsetDate,
     data: FormDataRecalculated): Validated[GformError, Unit] =
-    validateInputDate(fieldValue, fieldValue.id, fieldValue.errorMessage, data).andThen(
+    validateInputDate(fieldValue, fieldValue.id, fieldValue.errorMessage, data, None).andThen(
```

Comment:
We seem to be calling `validateInputDate` with None at the end quite a lot - maybe and only maybe we could consider a default argument (not ideal I would normally discourage) or an overloaded function or else

Reply:
changes were resolved